frontend/src/components/global/CTA.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+class Contact extends Component {
```

Comment:
can we change this component name to another name like "CallToAction" because we have another component named "Contact"?

Reply:
This has been updated